unittest/test_load.py

```diff
@@ -57,6 +57,9 @@ def test_anymal_kinova(self):
     def test_baxter(self):
         self.check("baxter", 19, 19)
 
+    def test_falcon_bravo7(self):
+        self.check("falcon_bravo7", 13, 12)
```

Comment:
If we need to add continuous joints, then this needs to be updated.

Reply:
Done